src/main/java/frc/robot/IMU/BNO055.java

```diff
@@ -0,0 +1,776 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.IMU;
+
+import java.util.TimerTask;
+
+import edu.wpi.first.wpilibj.I2C;
+import edu.wpi.first.wpilibj.Timer;
+public class BNO055 {
+	//Tread variables
+	private java.util.Timer executor;
+	private static final long THREAD_PERIOD = 20; //ms - max poll rate on sensor.
```

Comment:
Keep this in mind, we're not going to be able to get updates if we're calling more than more than 50 Hz